### PR TITLE
remove direct calls to graph executor

### PR DESCRIPTION
**Pull request by eellison**
Removing direct calls to graph executor, which uses "evilDeprecatedBadCreateStackDoNotUse"

### CODE REVIEW

File: test/test_jit.py [hunk 1]
```diff
@@ -1022,10 +1018,12 @@ def foo(a, b):
             return a * b / (a - b) + b
         V = Variable
         a, b = V(torch.rand(1)), V(torch.rand(1))
-        ge = torch._C.GraphExecutor(foo, (a, b))
+        ge, _ = torch.jit.get_trace_graph(foo, (a, b))
+        m = torch.jit.ScriptModule()
+        m._create_method_from_graph("forward", ge.graph())
```
Comment (jamesr66a) [inline test/test_jit.py]: I'm changing this so we don't extract the graph and build a new module. Do you know if there are other places we do this?

Comment (eellison) [reply test/test_jit.py]: It's also done in assertExportImport

**Review by goldsborough — APPROVED** [review]
LGTM but ask @jamesr66a @zdevito @apaszke for approval

File: test/test_jit.py [hunk 2]
```diff
@@ -1022,10 +1018,12 @@ def foo(a, b):
...
         a, b = V(torch.rand(1), requires_grad=True), V(
             torch.rand(1), requires_grad=True)
-        r, = ge(a, b)
+        r, = m.forward(a, b)
```
Comment (apaszke) [inline test/test_jit.py]: Modules should never be called like `m.forward(a, b)`, but `m(a, b)`

**Review by goldsborough — APPROVED** [review]
looks good, no more evil

**Pull request closed without merging**